src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -457,28 +457,66 @@ private EmailWrapper generateFeedbackSessionEmailBaseForInstructors(
         List<EmailWrapper> emails = new ArrayList<EmailWrapper>();
 
         for (InstructorAttributes instructor : instructors) {
-            emails.add(generateFeedbackSessionClosedEmail(course, session, instructor.name, instructor.email));
+            emails.add(generateFeedbackSessionClosedEmailForInstructors(course, session, instructor.name, instructor.email));
         }
         for (StudentAttributes student : students) {
-            emails.add(generateFeedbackSessionClosedEmail(course, session, student.name, student.email));
+            emails.add(generateFeedbackSessionClosedEmailForStudents(course, session, student.name, student.email));
         }
 
         return emails;
     }
-
-    private EmailWrapper generateFeedbackSessionClosedEmail(CourseAttributes course,
+    
+    /**
+     * Generates the feedback session closed email for the students with the given {@code userName} and {@code userEmail}.
+     */
+    private EmailWrapper generateFeedbackSessionClosedEmailForStudents(CourseAttributes course,
             FeedbackSessionAttributes session, String userName, String userEmail) {
-        String template = EmailTemplates.USER_FEEDBACK_SESSION_CLOSED;
+    	
+    	String template = EmailTemplates.USER_FEEDBACK_SESSION_CLOSED;
+    	
         String subject = EmailType.FEEDBACK_CLOSED.getSubject();
-
+        
         String emailBody = Templates.populateTemplate(template,
-                "${userName}", SanitizationHelper.sanitizeForHtml(userName),
+        		"${userName}", SanitizationHelper.sanitizeForHtml(userName),
+        		"${instructorFragment}", "",
                 "${courseName}", SanitizationHelper.sanitizeForHtml(course.getName()),
                 "${courseId}", SanitizationHelper.sanitizeForHtml(course.getId()),
                 "${feedbackSessionName}", SanitizationHelper.sanitizeForHtml(session.getFeedbackSessionName()),
                 "${deadline}", SanitizationHelper.sanitizeForHtml(TimeHelper.formatTime12H(session.getEndTime())),
                 "${supportEmail}", Config.SUPPORT_EMAIL);
-
+        
+        EmailWrapper email = getEmptyEmailAddressedToEmail(userEmail);
+        email.setSubject(String.format(subject, course.getName(), session.getFeedbackSessionName()));
+        email.setContent(emailBody);
+        return email;
+    }
+    
+    /**
+     * Generates the feedback session closed email for the instructors with the given {@code userName} and {@code userEmail}.
+     */
+    private EmailWrapper generateFeedbackSessionClosedEmailForInstructors(CourseAttributes course,
```

Comment:
Take a look: this method and the previous is almost identical, with the exception of how `${instructorFragment}` is handled. See if you can suppress such repetition.